pkg/graph/schema.resolvers_system_intake_test.go

```diff
@@ -1274,7 +1275,8 @@ func (s *GraphQLTestSuite) TestUpdateRequestDetails() {
 				businessSolution: "My solution",
 				businessNeed: "My need",
 				currentStage:  "Just an idea",
-				needsEaSupport: false
+				needsEaSupport: false,
+				hasUiChanges: false,
```

Comment:
Could you add a test case that sets `true`, and maybe one that doesn't pass it at all (and should get a `nil` in the response, I'd imagine?

Reply:
Added those tests.